test/templates/nstemplatetiers/metadata.yaml

```diff
@@ -1,6 +1,10 @@
-advanced-code: "123456a"
-advanced-dev: "123456b"
-advanced-stage: "123456c"
-basic-code: "123456d"
-basic-dev: "123456e"
-basic-stage: "1234567" # here, a number string
```

Comment:
didn't we forget to test the number string?

Reply:
done in 780068f